### PR TITLE
M3-59 M3-54 M3-60 M3-61 M3-62 Putting er all together.

### PR DESCRIPTION
**Pull request by rjstires**
You might want to use production services, otherwise you'll only see one region coming back from the API.

### CODE REVIEW

**Comment by alioso** [comment]
Applied all necessary styling and adjusted some default form styles

You may notice some absolute positioning on the strength indicator and this to amount for the fact that we don't want the content underneath to shift when the strength indicator pops up. This solution is better than to add empty markup IMO. Example updated in the story to reflect that

File: src/features/linodes/LinodesCreate/AddonsPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+type CombinedProps = Props & WithStyles<ClassNames>;
+
+class AddonsPanel extends React.Component<CombinedProps> {
+  render() {
+    const { classes, handleChange } = this.props;
+    return (
+      <Paper className={classes.root}>
+      <div className={classes.inner}>
```
Comment (asauber) [inline src/features/linodes/LinodesCreate/AddonsPanel.tsx]: indentation

File: src/features/linodes/LinodesCreate/AddonsPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                <Typography variant="body2">
+                We need copy! We need copy! We need copy! We need copy! We need copy! We need copy!
+                We need copy! We need copy! We need copy! We need copy! We need copy! We need copy!
+                </Typography>
+              </Grid>
+            </Grid>
+          </Grid>
+        </Grid>
+      </div>
+    </Paper>
```
Comment (asauber) [inline src/features/linodes/LinodesCreate/AddonsPanel.tsx]: indentation

File: src/features/linodes/LinodesCreate/LinodesCreate.tsx [hunk 3]
```diff
@@ -30,46 +37,129 @@ interface Props {
...
+        ...response,
+        data: response.data.map((type) => {
+          const { memory, vcpus, disk, price: { monthly, hourly } } = type;
+          const mem = memory / 1024;
+          return ({
+            ...type,
+            heading: `Linode ${ mem }G`,
+            subHeadings: [
+              `$${monthly}/mo ($${hourly}/hr)`,
+              `${vcpus} CPU(s), ${ Math.floor(disk / 1000) }GB Storage, ${mem}G RAM`,
```
Comment (asauber) [inline src/features/linodes/LinodesCreate/LinodesCreate.tsx]: we can use the functions in `presentation.ts` for these

**Pull request merged**